### PR TITLE
feat: add kotlin for backend as trial language

### PR DESCRIPTION
**Pull request by infantesimone**
The goal of this PR is to add Kotlin for backend as trial language.

- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

**Review by uolter — APPROVED** [review]

**Comment by gunzip** [comment]
Thank you @infantesimone for your proposal to include Kotlin in our technology radar. While I appreciate the initiative, I have some reservations about placing it in the "Adopt" category for the following reasons:

1. Limited usage scope: currently, Kotlin's usage seems to be confined to a single team within our organization.
1. Lack of widespread expertise: we do not have widespread expertise in Kotlin across the company.
1. Past experience with "niche" languages: we've experienced scenarios where projects developed in languages without widespread expertise within the company had to be rewritten in more "familiar" languages.
1. Technology Radar's purpose: the "adopt" category is meant for technologies that are advisable for new projects due to their proven reliability and integration into our workflow; this includes technologies in which **we plan to invest** as a company, such as in tooling and shared libraries. At this point, do we foresee having sufficient resources and commitment from the Kotlin devs to support such investments?

Given these points, I believe we should proceed with caution before considering Kotlin for the "Adopt" quadrant. It might be more appropriate to place it in a quadrant that reflects our current level of engagement and investment in this technology.

Looking forward to someone else thoughts on this!

**Comment by iwoak** [comment]
Thanks for your proposal @infantesimone, that's the right mood! 
I agree with @gunzip, we should consider the assess or trial ring.

**Comment by francescopersico** [comment]
I want to add another POV.
Kotlin is the standard we use to develop **native mobile application** (smartpos) and also **native modules** for React Native.

ref.
https://github.com/pagopa/io-react-native-crypto
https://github.com/pagopa/io-react-native-login-utils

**Comment by gunzip** [comment]
We can have two distinct entries (Kotlin for mobile -> adopt, Kotlin for backend -> trial)

**Comment by lucacavallaro** [comment]
agree with @gunzip, we should separate use cases

**Comment by infantesimone** [comment]
OK, I agree. 
Maybe change the title to "Kotlin for backend" (trial)? and as references only the repos on which it was used for backend

**Comment by gunzip** [comment]
Yep!

**Review by iwoak — APPROVED** [review]

File: radar/languages-and-frameworks/kotlin.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Kotlin for backend"
+ring: trial
+quadrant: "languages-and-frameworks"
+tags: [kotlin]
```
Comment (gunzip) [inline radar/languages-and-frameworks/kotlin.md:5]: I'd add some more tag (ie. api, backend, ...)

**Review by gunzip — APPROVED** [review]
See comments.

**Pull request merged**